pkg/daemon/metrics.go

```diff
@@ -348,8 +351,8 @@ func extractSummaryMetrics(configName, processName, output string) (iface string
 		fields = append(fields, "") // Making space for the new element
 		//  0             1     2
 		//ptp4l.0.config rms   53 max   74 freq -16642 +/-  40 delay  1089 +/-  20
-		copy(fields[2:], fields[1:])                       // Shifting elements
-		fields[1] = masterOffsetIface.get(configName).name // Copying/inserting the value
+		copy(fields[2:], fields[1:])                        // Shifting elements
+		fields[1] = masterOffsetIface.get(configName).alias // Copying/inserting the value
```

Comment:
Can you follow through this , There is a function getEvenSource which is driven by interface full name , usually I think master and CLOCK_REALTIME. Just want to make sure we are not missing anything

Reply:
The caller of `GetEventSource` calls `getByAlias(...).name` first so shouldn't be an issue.